### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by renercrows**
### Milestone 1 Webpage

- Created the index for the page, then created the about me page
- Made some CSS to the page with different media queries, one for mobile, another for the standard desktop resolution, and two different ones for bigger screens.
- Made the JavaScript for the hamburger icon, and the nav menu, also made the cards of the artists with DOOM manipulating objects.

Link for the Loom video:
https://www.loom.com/share/c5da94c1cd854f1f8177df58d2651c27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (DeliceLydia) [inline README.md]: - [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+👤 **Maximiliano Ungredda**
+
+- GitHub: [@renercrows](https://github.com/renercrows)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```
Comment (DeliceLydia) [inline README.md]: - [ ] Please deploy your page on GitHub and add a link in the README

File: about.html [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+                    <div class="footer-info">
+                        <p>2009 ThunderWold USA. Some Rights Reserved.<br /><br />
+                        Rights are reserved for the ThunderWolf concerts.
+                        <br /><br />
+                        Not allow to share any information about logistic or any strategy of the concert.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </footer>
```
Comment (DeliceLydia) [inline about.html:132]: - [ ] I suggest you give similar spacing, borders, shadows, and paddings to the original design.
design

File: script.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    info: 'One of the top pop singuers',
+  },
+
+  {
+    img: 'images/singuer6.png',
+    name: 'Maroon 5',
+    title: 'Pop Artist',
+    info: 'One of the top pop bands',
+  },
+];
```
Comment (DeliceLydia) [inline script.js:59]: - [ ] Featured speakers section spacing, borders, shadows, and paddings than the original design
speaker

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @renercrows, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there! Kudos to the:
- good looking UI
- Followed git workflow


## Suggested changes
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by andersonlebon — APPROVED** [review]
## Hi @renercrows 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**